src/Serializer/Normalizer/PhoneNumberNormalizer.php

```diff
@@ -85,6 +85,10 @@ public function supportsNormalization($data, $format = null)
      */
     public function denormalize($data, $class, $format = null, array $context = [])
     {
+        if (null === $data) {
```

Comment:
Should be in another PR I think?

Reply:
You're right, it's a test I forgot to remove